src/styles/blocks/stars.scss

```
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+
+  & :not(:last-child) {
```

```suggestion
  &:not(:last-child) {
```
and move into &__star

---

If I remove this space but the function does not work for me